app/src/main/res/values/strings.xml

```diff
@@ -104,7 +104,7 @@
     <string name="settings_category_player_title">Player</string>
     <string name="settings_category_player_behavior_title">Behavior</string>
     <string name="settings_category_video_audio_title">Video &amp; Audio</string>
-    <string name="settings_category_history_title">History &amp; Cache</string>
+    <string name="settings_category_history_title">History</string>
```

Comment:
I think that was called "History & Cache" because there is an option to clean stored metadata. When we rename this category, we should also move this option. Otherwise it's hard to find.

Reply:
I agree. I wouldn't move it I would keep the string as it is.

Reply:
I reverted it